feature-libs/cart/base/root/models/cart.model.ts

```diff
@@ -193,3 +194,13 @@ export enum CartValidationStatusCode {
   PRICING_ERROR = 'pricingError',
   UNRESOLVABLE_ISSUES = 'unresolvableIssues',
 }
+
+/**
+ * Base Cart options, used for extensibility.
+ */
+export type BaseCartOptions<T> = BaseUserOptions<T & { cartId: string }>;
+
+export interface AddEntryOptions {
+  productCode: string;
+  quantity?: number;
```

Comment:
do we want to add `posNam`e here, or augmented in bopis?

Reply:
I say augment